app/profile/ProfileSettings.tsx

```diff
@@ -0,0 +1,148 @@
+import React from "react";
+import { SafeAreaView } from "react-native-safe-area-context";
+import { View, Text, Pressable, TextInput } from "react-native";
+import { useState } from "react";
+import { Image, Alert } from "react-native";
+import * as ImagePicker from "expo-image-picker";
+import NoProfileImg from "../../assets/icons/profile/no-profile-img.svg";
+import CameraIcon from "../../assets/icons/profile/camera-icon.svg";
+
+const ProfileSettings = () => {
+  const [profileImage, setProfileImage] = useState<string | null>(null);
+
+  const uploadProfileImage = async () => {
+    const { status } = await ImagePicker.requestMediaLibraryPermissionsAsync();
+
+    if (status === "denied") {
+      Alert.alert(
+        "Permission Required",
+        "Permission to access the media library is required. Please enable it in settings.",
+        [
+          {
+            text: "Cancel",
+            style: "cancel",
+          },
+          {
+            text: "Try Again",
+            onPress: async () => {
+              const newPermissionResult =
+                await ImagePicker.requestMediaLibraryPermissionsAsync();
+              if (newPermissionResult.granted) {
+                launchImagePicker();
+              } else {
+                Alert.alert(
```

Comment:
Instead of alert, use `Toast` by `react-native-toast-message`. You may see examples from `login` page